src/bp/core/user-code/index.ts

```diff
@@ -4,3 +4,5 @@ export * from './action-server/tasks-repository'
 export * from './action-service'
 export * from './hook-service'
 export * from './vm'
+export * from './hints/hints-service'
+export * from './hints/hints-router'
```

Comment:
I'm not sure I understand why the hints-service is found in the user-code folder? Why not put it directly under core?

Reply:
Because it scans user-generated files to extract variable names to provide "hints" (or auto complete) in the code editor, therefore it's related to user-code :)

Reply:
Ahhh! Perfect, thanks for the clarification!!